### PR TITLE
Add light, medium, and heavy starting units to TS.

### PR DESCRIPTION
**Pull request by phrohdoh**

### CODE REVIEW

File: mods/ts/rules/world.yaml [hunk 1]
```diff
@@ -168,12 +168,52 @@ World:
...
+		Class: light
+		ClassName: Light
+		Races: nod
+		BaseActor: mcv
+		SupportActors: e1,e1,e1,e3,e3
+		InnerSupportRadius: 3
+		OuterSupportRadius: 5
+	MPStartUnits@med.gdi:
+		Class: medium
+		ClassName: Meduim
```
Comment (reaperrr) [inline mods/ts/rules/world.yaml]: Typo detected :)

File: mods/ts/rules/world.yaml [hunk 2]
```diff
@@ -168,12 +168,52 @@ World:
...
+		Races: gdi
+		BaseActor: mcv
+		SupportActors: e1,e1,e2,jumpjet,smech
+		InnerSupportRadius: 3
+		OuterSupportRadius: 5
+	MPStartUnits@med.nod:
+		Class: medium
+		ClassName: Meduim
```
Comment (reaperrr) [inline mods/ts/rules/world.yaml]: Same here.

File: mods/ts/rules/world.yaml [hunk 3]
```diff
@@ -168,12 +168,52 @@ World:
...
+		OuterSupportRadius: 5
+	MPStartUnits@heavy.nod:
 		Class: heavy
-		ClassName: Heavy Support
-		Races: gdi,nod
+		ClassName: Heavy
+		Races: nod
 		BaseActor: mcv
-		SupportActors: e1,e2,e3,cyborg,cyc2,jumpjet,umagon,ghost,mmch,smech,hmec,bggy,bike,art2,ttnk,hvr,sonic,subtank,repair,4tnk
+		SupportActors: e1,e3,bggy,bike,bike
```
Comment (reaperrr) [inline mods/ts/rules/world.yaml]: I'd replace one bike with a tick tank.

File: mods/ts/rules/world.yaml [hunk 4]
```diff
@@ -168,12 +168,52 @@ World:
...
+		BaseActor: mcv
+		SupportActors: e1,e1,e3,cyborg,bggy
+		InnerSupportRadius: 3
+		OuterSupportRadius: 5
+	MPStartUnits@heavy.gdi:
+		Class: heavy
+		ClassName: Heavy
+		Races: gdi
+		BaseActor: mcv
+		SupportActors: e1,e1,e2,smech,mmch
```
Comment (reaperrr) [inline mods/ts/rules/world.yaml]: I'd replace 1 e1 with an e2, that should roughly offset the tick tank for nod.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by reaperrr** [comment]
:+1:

**Comment by UberWaffe** [comment]
For GDI and NOD medium support: GDI gets Jumpjet (Air unit) and Nod gets only 1 Rocket infantry (e3). If I remember correctly a jumpjet-vs-e3 1-vs-1 the jumpjet will win.
Should just test this. (Not really a problem, since e3 only requires a hand of Nod to be built.)

As a complete sidenote idea: How about a fast-start where instead of combat units you start with a harvester or two extra?

**Pull request merged**